src/content/about/structure/index.md

```diff
@@ -27,10 +27,28 @@ DMET is divided into multiple portfolio teams, each with a specific focus. The d
 
 ## Portfolio
 
-Our portfolio is a collection of work that is managed to achieve goals and deliver a strategy. To find out more about each portfolio team, please follow these links:
+Our portfolio is our collection of work that is managed to achieve goals and deliver a strategy. To find out more about each portfolio team, please follow these links:
 
-- [Centre for Excellence (CfE)](https://moj-analytical-services.github.io/dmet-cfe/)
-- [HMPPS](https://justiceuk.sharepoint.com/:p:/r/sites/DataEngineeringDataScience/Shared%20Documents/General/1.%20Data%20%26%20Analytics%20Engineering%20Hub/3.%20Prison,%20Probation%20%26%20EM/hmpps_posters.pptx?d=w17120a8f29f2469cbfbe85be1a7049d2&csf=1&web=1&e=jX6jC7)
-- HMCTS - Under construction
-- [Corporate and Programmes](https://justiceuk.sharepoint.com/:p:/s/DMETCorporate/EWYWM-dXQtlHguKF-Dnx8jUB8MsLZXWfamZ93GLF9cdhwA?e=S7fQ7s)
-- Centre for Enablement - Under construction
+#### HM Prison and Probation Service (HMPPS)
+
+The HMPPS domain in Data and Analytics engineering, are responsible for providing users on the Analytical Platform access to data covering Prisons and Probation data for analytical purposes.
+
+Internal only [link](https://justiceuk.sharepoint.com/:p:/r/sites/DataEngineeringDataScience/Shared%20Documents/General/1.%20Data%20%26%20Analytics%20Engineering%20Hub/3.%20Prison,%20Probation%20%26%20EM/hmpps_posters.pptx?d=w17120a8f29f2469cbfbe85be1a7049d2&csf=1&web=1&e=jX6jC7) to Show and Tell posters.
+
+#### HM Courts and Tribunals Service (HMCTS)
+
+The HMCTS domain in Data and Analytics engineering, are responsible for providing users on the Analytical Platform access to data covering Civil, Family, Tribunals, Magistrates, and Crown court data for analytical purposes.
+
+#### Corporate and Programmes
+
+The Corporate & Programmes domain in Data and Analytics engineering, are responsible for providing users on the Analytical Platform access to data covering Finance, People and Commercial, as well as the BOLD and Data First programmes, for Analytical Purposes.
+
+Internal only [link](https://justiceuk.sharepoint.com/:p:/s/DMETCorporate/EWYWM-dXQtlHguKF-Dnx8jUB8MsLZXWfamZ93GLF9cdhwA?e=S7fQ7s) to Show and Tell posters.
+
+#### Centre for Excellence (CfE)
+
+[Slide deck summarising CfE](https://moj-analytical-services.github.io/dmet-cfe/).
+
+#### Centre for Enablement
+
+Under construction!
```

Comment:
There is a question about if we should include it if the CfE aims are not summarised.

Reply:
Unlikely this will be populated until after Phil's TRA as he's technically no longer in the team. Probably worth removing for now.

Reply:
OK, lets remove the section for now.